deployment/network-operator/values.yaml

```diff
@@ -32,28 +33,37 @@ sriovNetworkOperator:
 
 # Node Feature discovery chart related values
 node-feature-discovery:
-  image:
-    pullPolicy: IfNotPresent
-  nodeFeatureRule:
-    createCRD: false
-  master:
-    instance: "nvidia.networking"
+  enableNodeFeatureApi: true
   worker:
+    serviceAccount:
+      name: node-feature-discovery
+      # disable creation to avoid duplicate serviceaccount creation by master spec below
+      create: false
     tolerations:
-      - key: "nvidia.com/gpu"
-        operator: "Equal"
-        value: "present"
-        effect: "NoSchedule"
+    - key: "node-role.kubernetes.io/master"
```

Comment:
why do we need this (master toleration) as default ?
for usecases where we need worker to run on master, deployer can override it in values.yaml during deployment

also, if there is master, there should be also control-plane toleration (newer clusters deplyoed with kubeadm have the latter)

Reply:
I am always thinking -2 or -3 releases down the road. I know this might sound like something the deployer could add manually, but I find my self deploying the GPU-Operator on single node deployments every week. so it has become a default to add the toleration for master (single node)

Reply:
ack,
can you add also for control-plane label ? as done below.

```
      - key: "node-role.kubernetes.io/master"
        operator: "Exists"
        effect: "NoSchedule"
      - key: "node-role.kubernetes.io/control-plane"
        operator: "Exists"
        effect: "NoSchedule"
```

newer kubeadm does not label nodes with master but rather with control-plane.